### ISSUE TITLE
virtual environment created via handoff is broken

### ISSUE THREAD

**Issue by daigotanaka** [anelendata/handoff#25]
Mac OS 10.15.5

```
$ which python3
/usr/bin/python3
```

```
python3 --version
Python 3.7.3
```

```
git checkout https://github.com/anelendata/handoff/commit/5cb822db0ad5d9f0e43fa6024bfcca86b30c7212
python3 -m venv venv
source venv/bin/activate
cd handoff
pip install -e .
handoff quick_start start
cd projects
handoff -p 03_exchange_rates -w workspace workspace install
```

You will see errors:
```
tap-exchangeratesapi 0.1.1 has requirement singer-python==5.3.3, but you'll have singer-python 2.1.4 which is incompatible.
```
...which seems to be conflicting with target-csv requirements...

Now:
```
deactivate
source workspace/proc_01/bin/activiate
python
```

```
dyld: Library not loaded: @executable_path/../Python3
  Referenced from: /Users/.../workspace/proc_01/bin/python
  Reason: image not found
Abort trap: 6
```

**Comment by daigotanaka** [comment]
It seems to be related to Python update with home-brew. Potential solutions:
- https://mattbanderson.com/so-you-hosed-your-mac-os-python-install/
- https://medium.com/@iantonioribeiro/dyld-library-not-loaded-executable-path-python-d044296efe47
It may also be related:
- https://stackoverflow.com/questions/47629570/how-to-resolve-dyld-library-not-loaded-executable-path-error

**Comment by daigotanaka** [comment]
Actually,
```
builder = pyvenvx.ExtendedEnvBuilder(symlinks=True) 
```
in handoff/core/admin.py:Ln33 did the trick. Now testing this with Linux (Ubuntu) and Docker environment...

**Comment by daigotanaka** [comment]
fixed in 0.2.0b0
